projects/angular-cld/src/lib/cloudinary-image.component.spec.ts

```diff
@@ -649,7 +649,35 @@ describe('CloudinaryImage', () => {
       expect(img.getAttribute('style')).toEqual(jasmine.stringMatching('opacity: 0; position: absolute;'));
     });
   });
+  describe('cl-image with placeholder and html style', () => {
+    @Component({
+      template: `<div style="margin-top: 4000px"></div>
+      <cl-image loading="lazy" public-id="sample" width="500" crop="fit" style="max-height: 100%">
+        <cl-placeholder type="blur"></cl-placeholder>
+      </cl-image>`
+    })
+    class TestComponent {}
+
+    let fixture: ComponentFixture<TestComponent>;
+    let des: DebugElement[];  // the elements w/ the directive
+    let testLocalCloudinary: Cloudinary = new Cloudinary(require('cloudinary-core'),
+      { cloud_name: '@@fake_angular2_sdk@@', client_hints: true } as CloudinaryConfiguration);
+    beforeEach(() => {
+      fixture = TestBed.configureTestingModule({
+        declarations: [CloudinaryTransformationDirective, CloudinaryImage, TestComponent, CloudinaryPlaceHolder],
+        providers: [{ provide: Cloudinary, useValue: testLocalCloudinary }]
+      }).createComponent(TestComponent);
+
+      fixture.detectChanges(); // initial binding
+      // all elements with an attached CloudinaryImage
+      des = fixture.debugElement.queryAll(By.directive(CloudinaryImage));
+    });
 
+    it('should have style opacity and position when style is passed', () => {
```

Comment:
Maybe also add a check that the style that was passed exists? (check that we didn't discard the passed value)

Reply:
alright will update